### PR TITLE
Responsive version

### PR DESCRIPTION
**Pull request by Amrit131**
# Desktop version
### Few functionalities that we added are -:
- Used media query to adjust the website according to screen sizes.
- Added styling to all sections of the website according to  desktop version.
- Positioned element using Grid and Flexbox according to desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,55 +14,129 @@
...
-            <div id="bg-container" class="socials-container">
-                    <img id="bg-img" src="images/Groupbackground.png" alt="Background card">  
+            <div id="col-1">
+                <div id="first-content">
+                    <h1 id="headline">Hey There.<br>I'm Amrit</h1>
+                    <h2 id="title-line">I'am a Software Developer</h2>
+                    <p id="para">I can help you build a product, feature or website Look through some of my work <br> and experience! If you like what you see <br> and have a project you need coded, don't hestiate to contact me.</p>
+                </div>
+                <ul class="socials-container">
+                    <li class="socials">
```
Comment (aimalamiri) [inline index.html:36]: Good job, but as you know it's essential to follow the exact Figma template, so please make sure to add some space from left to your social icons.

image

File: index.html [hunk 2]
```diff
@@ -14,55 +14,129 @@
...
+                <ul id="menu-list">
+                    <li><a class="anchor" href="#">Portfolio</a></li>
+                    <li><a class="anchor" href="#">About</a></li>
+                    <li><a class="anchor" href="#">Contact</a></li>
+                    <li><a class="anchor" href="#"><img src="images/mail.png" alt="Mail image"></a></li>
+                </ul>
+            </div>
         </nav>
     </header>
     <main id="content">
```
Comment (aimalamiri) [inline index.html:27]: It looks nice, but kindly make sure to decrease the height of this section, it'll make it much more beautiful.

File: index.html [hunk 3]
```diff
@@ -14,55 +14,129 @@
...
-                    <li class="btn"><button>html</button></li>
-                    <li class="btn"><button>bootstrap</button></li>
-                    <li class="btn"><button>Ruby</button></li>
-                </ul>
-                <div id="btn-container">
-                    <button id="btn">See Project</button>
+            <div id="works-container">
+                <div id="header2-container">
+                    <h2 id="header-2">My Recent Works</h2>
+                    <hr id="line">
```
Comment (aimalamiri) [inline index.html:62]: It's almost done! but please make sure to fix the extra space between the line and the text.

image

File: index.html [hunk 4]
```diff
@@ -14,55 +14,129 @@
...
                 </div>
-            </div>
+                <div id="gymfit-container">
+                    <img id="gymfit-img" src="images/gym-fit.png" alt="Gym Fit Image">
```
Comment (aimalamiri) [inline index.html:65]: Kindly make sure to follow the exact sizing properties from the Figma design, and fix the width of your image. 

image

File: index.html [hunk 5]
```diff
@@ -174,18 +250,63 @@ <h5 id="header-5">About me</h5>
...
+                </div>
+                <div class="sub-list">
+                  <div class="img-shape">
+                    <img src="images/2.png" alt="second shape">
+                    <h5 class="head" >Frameworks</h5>
+                  </div>
+                  <ul class="list">
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
```
Comment (aimalamiri) [inline index.html:274]: It seems you limited the width for this section so please make sure to update this part of your page.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Amrit131, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
Your project looks great generally, but you still have some minor sizing and spacing issues that you need to work on.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Amrit131) [reply index.html:36]: # Implemented

1st

File: index.html (hunk 3 above)
Comment (Amrit131) [reply index.html:62]: # Implemented

2nd

**Review by ichala — APPROVED** [review]
Hi @Amrit131  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- No Linters Error✔️  
- Changes Implemented ✔️


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Amrit131) [reply index.html:65]: # Implemented

3rd

File: index.html (hunk 5 above)
Comment (Amrit131) [reply index.html:274]: # Implemented

4th

**Comment by Amrit131** [comment]
Thank you for the good news.

**Pull request merged**